packages/components/ui/src/gauges/screen/README.md

```diff
@@ -0,0 +1,81 @@
+# Screen Gauge Component
+
+You can use this component to measure screen features reactively.
+
+## `screen` store
+
+This component exports a store like the below:
+
+```js
+{
+	classes: 'small medium large xLarge',
+	display: {
+		aspectRatio: 1.1550632911392404,
+		height: 948,
+		orientation: {
+			angle: 0,
+			onchange: null,
+			type: 'landscape-primary'
+			__proto__: ScreenOrientation
+		},
+		pixelRatio: 2,
+		width: 1095,
+	},
+	glyph: {
+		height: 24,
+		width: 8.865384615384615,
+	},
+	orientation: {
+		landscape: true,
+		portrait: false,
+	},
+	size: {
+		large: true,
+		medium: true,
+		small: true,
+		xLarge: true,
+		xSmall: false,
+	},
+	text: {
+		height: 39.5,
+		width: 123.51409978308027
+	}
+}
+```
+
+## Usage
+
+### Render `<ScreenGauge/>`
+
+In a component of your app, instantiate this gauge.
+In a Sapper app this would usually be a `_layout.svelte` file.
+
+```svelte
+<script>
+	import ScreenGauge from '@svizzle/ui/src/gauges/ScreenGauge.svelte';
+
+	const isDev = process.env.NODE_ENV === 'development';
+</script>
+
+<ScreenGauge
+	{isDev}
+	bands={[60, 82, 100, 120]}
+/>
+```
+
+### Use `screen`
+
+In other components of your app, `import` the `screen` store from `ScreenGauge.svelte` and use as you see fit.
+
+```svelte
+<script>
+	import {screen} from '@svizzle/ui/src/gauges/ScreenGauge.svelte';
+
+	console.log($screen);
+</script>
+
+{#if $screen.size.medium && $screen.orientation.landscape}
+	<p>Medium (landscape)</p>
+{/if}
+/>
```

Comment:
What do you think about expanding the example to include some class selectors?

Reply:
Maybe adding that they allow for progressive enhancements starting from `small` as default